embedded-bins/etcd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13-alpine AS build
+FROM golang:1.15-alpine AS build
```

Comment:
it seems the problem is related to the line 9 which is missing either -mod=mod or -mod=readonly
We need to investigate whether building this way would break etcd in anyway.

For me it worked when I passed this:
```
RUN  CGO_ENABLED=0 go build -ldflags="-w -s" -o etcd -mod=mod
```

Reply:
updated. thanks you @jasmingacic